app/views/forum/topic.scala

```diff
@@ -158,7 +158,7 @@ object topic {
                   if (topic.hidden) "Feature" else "Un-feature"
                 )
               ),
-            canModCateg || ctx.me.exists(topic.isAuthor) option
+            canModCateg || topic.isUblog && ctx.me.exists(topic.isAuthor) option
```

Comment:
could you add parens? It's hard to guess what this does without knowing precedence rules here